pyproject.toml

```diff
@@ -63,9 +63,12 @@ simplejson = ">=3.17.0"
 # Optional dependencies for markup extra
 anymarkup-core = { version = ">=0.8.0", optional = true }
 xmltodict = { version = ">=0.12.0", optional = true }
+# Optional dependencies for shell functions integration tests
+pytest-container = {version = "*", optional = true}
```

Comment:
This is not how to do it. You need to define a new dependency group and move it to that.

Reply:
ok moved it into a new dep group

Reply:
data.group.test must not contain {'optional_dependencies'} properties